rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -286,7 +286,6 @@ echo " Done"
 # Do this ONLY at first install
 if [ $1 -eq 1 ]
 then
-		echo -e '# This sources the configuration file needed by Rudder\n. /etc/sysconfig/rudder-apache' >> /etc/sysconfig/apache2
```

Comment:
This is already done just a few lines below, with a safeguard to avoid adding it twice